sql/load_demo_data.sql

```diff
@@ -1807,7 +1901,9 @@ CREATE TABLE `oh_grouppermission` (
   `GP_CREATED_DATE` datetime DEFAULT NULL,
   `GP_LAST_MODIFIED_BY` varchar(50) DEFAULT NULL,
   `GP_LAST_MODIFIED_DATE` datetime DEFAULT NULL,
-  PRIMARY KEY (`GP_ID`)
+  PRIMARY KEY (`GP_ID`),
+  KEY `FK_GROUPPERMISSION_CREATED_BY_IDX` (`GP_CREATED_BY`),
+  KEY `FK_GROUPPERMISSION_LAST_MODIFIED_BY_IDX` (`GP_LAST_MODIFIED_BY`)
```

Comment:
This table is missing two contraints:
<pre>
CONSTRAINT `FK_GROUPPERMISSION_CREATED_BY` FOREIGN KEY (`GP_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION,
CONSTRAINT `FK_GROUPPERMISSION_LAST_MODIFIED_BY` FOREIGN KEY (`GP_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION
</pre>

Reply:
In addition the demo data load has the table as MyISAM whereas the multiple step version has it as InnoDB

Reply:
Yeah, MyISAM causes the FK to be ignored, although specified in the step_95.
Added `ALTER TABLE OH_GROUPPERMISSION ENGINE = INNODB, CONVERT TO CHARACTER SET utf8;` to step_95.
Thanks!

Reply:
BTW, I believe this is true of `oh_malnutritioncontrol` and `oh_permissions`; step 95 changed the engine type but the corresponding changes were not made to demo data.

Reply:
I think the main problem is that we don't have a rigid process to update demo data, so sometimes it could end up unalligned, especially during developments of the specific feature

Reply:
Fixed, please check again. Thanks!

Reply:
could you past here the missing lines please?

Reply:
<pre>
CONSTRAINT `FK_GROUPPERMISSION_CREATED_BY` FOREIGN KEY (`GP_CREATED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION,
 CONSTRAINT `FK_GROUPPERMISSION_LAST_MODIFIED_BY` FOREIGN KEY (`GP_LAST_MODIFIED_BY`) REFERENCES `oh_user` (`US_ID_A`) ON DELETE NO ACTION ON UPDATE NO ACTION
</pre>